src/components/Person/Person.jsx

```diff
@@ -1 +1,31 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+      {age && (
+        <p className="Person__age">
+          {`I am ${age}`}
+        </p>
+      )}
+      <p className="Person__partner">
+        {isMarried ? (
+          `${partnerName} is my ${sex === 'f' ? 'husband' : 'wife'}`
+        ) : (
+          'I am not married'
+        )}
+      </p>
```

Comment:
When you use ternary operator, please use formatting, each operand must start from a new line

Reply:
Fix comments of your mate